generated/nidcpower/nidcpower/session.py

```diff
@@ -4682,6 +4682,36 @@ def export_attribute_configuration_file(self, file_path):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def create_advanced_sequence(self, sequence_name, property_names, set_as_active_sequence=True):
+        '''create_advanced_sequence
+
+        Test.
+
+        Note: Test.
+
+        Args:
+            sequence_name (str): Name of sequence.
+
+            property_names (list of str): Sequence.
+
+            set_as_active_sequence (bool): Specifies that this current sequence is active.
+
+        '''
+        # First we need to get all possible properties we might be setting. The way the NI-DCPower C API is designed,
+        # we need to know this upfront in order to call `niDCPower_CreateAdvancedSequence`. In order to find the attribute
+        # ID of each property, we look at the member Attribute objects of Session.
+        attribute_ids_used = set()
+        for prop in property_names:
+            if prop not in Session.__base__.__dict__:
+                raise KeyError('{0} is not an property on the nidcpower.Session'.format(prop))
+            if not isinstance(Session.__base__.__dict__[prop], _attributes.Attribute) and not isinstance(Session.__base__.__dict__[prop], _attributes.AttributeEnum):
+                raise TypeError('{0} is not an attribute type: {1}'.format(prop, type(Session.__base__.__dict__[prop])))
```

Comment:
"attribute" as a term should never used in customer facing Python code.
"is not a valid property" would work imo